l10n_co_geo/__openerp__.py

```diff
@@ -0,0 +1,120 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution
+#    This module copyright (C) 2015 DevCO Colombia SAS
+#    (http://devco.co).
+#
+##############################################################################
+
+{
+    'name': 'Colombian Geodata',
+    'version': '1.0',
+    'author': 'DevCO Colombia SAS',
+    'maintainer': 'DevCO Colombia',
+    'website': 'http://devco.co',
+    'license': 'AGPL',
+    'category': 'Localization',
+    'summary': 'such as cities, departments and some improvements on displaying them.',
+    'description': """
+
+Colombian Geodata
```

Comment:
Move this to a README.rst file and follow latest template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst